Lesson4.2/src/com/home/model/MilitaryUnit.java

```diff
@@ -59,9 +63,12 @@ public int getNumberOfFreePlaces() {
         return recruitList;
     }
 
-    public void printAllRecruits() {
+    public void printRecruitsInfo() {
+        recruitList.sort(new SortByLastName());
+        System.out.println("Military unit № " + unitId + ":");
         for (Person recruit : recruitList) {
-            System.out.println(recruit.getName() + ", "
+            System.out.println(recruit.getName() + " "
```

Comment:
please refactor it a bit to print a numbered list of recruits, like
1. Ivan Ivanov
2. Vlasov Valerii
...